### PR TITLE
Allow verification_merchant_account_id to be passed in directly

### PR DESCRIPTION
**Pull request by arvanasse**
To ceate/update credit card

### CODE REVIEW

File: lib/conduit/braintree/actions/update_credit_card.rb [hunk 1]
```diff
@@ -27,6 +27,21 @@ def perform_request
...
+
+    # Request verification when the card is
+    # stored, using the merchant account id
+    # if one is provided
+    #
+    def whitelist_options
+      @options[:options] ||= {}.tap do |h|
+        h[:verify_card] = @options.fetch(:verify_card, true)
+        @options.delete(:verify_card)
+        if @options.key?(:verification_merchant_account_id)
```
Comment (dougperez) [inline lib/conduit/braintree/actions/update_credit_card.rb]: Do we need to allow ```verification_merchant_account_id``` in the options_attributes for the update cc action??

**Comment by dougperez** [comment]
:+1: Just had the one question.

File: lib/conduit/braintree/actions/update_credit_card.rb (hunk 1 above)
Comment (arvanasse) [inline lib/conduit/braintree/actions/update_credit_card.rb]: Yes, good catch.

**Comment by arvanasse** [comment]
Merged #25

**Pull request merged**